src/client.js

```diff
@@ -8,10 +8,8 @@ import storage from "./storage";
 import routes from "./routes";
 import sendEvent from "./google-tag-manager";
 import searchTest from "./search-test";
-import categories from "./constants/categories";
 
 window.searchTest = searchTest;
-window.categories = categories;
```

Comment:
Was this all there was to revert? It just enabled someone to be able to fiddle with some of the data on the client?